src/php/aspell.php

```diff
@@ -60,6 +60,12 @@
     // when splitting words.
     $text = preg_replace('/--+/u', ' ', $text);
 
+    // Get rid of e-mail addresses
+    $text = preg_replace("/[^@\s]*@[^@\s]*\.[^@\s]*/", ' ', $text);
+
+    // Get rid of URLs
+    $text = preg_replace("/[a-zA-Z]*[:\/\/]*[A-Za-z0-9\-_]+\.+[A-Za-z0-9\.\/%&=\?\-_]+/i", ' ', $text);
```

Comment:
I wonder if [filter_var](http://php.net/manual/en/function.filter-var.php), along with `FILTER_VALIDATE_EMAIL` and `FILTER_VALIDATE_URL` from [Validate filters](http://php.net/manual/en/filter.filters.validate.php) could be used to accomplish this?